test/mpi/init/session_psets.c

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) by Argonne National Laboratory
+ *     See COPYRIGHT in top-level directory
+ */
+
+#include "mpi.h"
+#include <stdio.h>
+#include "mpitest.h"
+
+/* This is adapted example 10.9 from MPI Standard 4.0 */
+
+static char use_pset_name[] = "SELF";
+
+int main(int argc, char *argv[])
+{
+    int errs = 0;
+    int i, n_psets, psetlen, rc, ret;
+    int valuelen;
+    int flag = 0;
+    char *pset_name = NULL;
+    char *info_val = NULL;
+    MPI_Session shandle = MPI_SESSION_NULL;
+    MPI_Info sinfo = MPI_INFO_NULL;
+    MPI_Group pgroup = MPI_GROUP_NULL;
+
+    rc = MPI_Session_init(MPI_INFO_NULL, MPI_ERRORS_RETURN, &shandle);
+    if (rc != MPI_SUCCESS) {
+        MTestPrintfMsg(1, "Could not initialize session, bailing out\n");
+        return -1;
+    }
```

Comment:
We need return `return MTestReturnValue(errs);` rather than `return -1;`

Reply:
Fixed.